user_templates/variables.cfg

```diff
@@ -176,8 +176,8 @@ variable_material_parameters: {
 ################################################
 ## this section is only considered if an ercf is installed and configured
 
-variable_ercf_unload_on_cancel_print: True
-variable_ercf_unload_on_end_print: True
+variable_ercf_unload_on_cancel_print: False
+variable_ercf_unload_on_end_print: False
```

Comment:
not sure if you want to invert the ercf functionality as it is now in klippain. The defaults of the config is the current behavior.

Reply:
In this case I think we should use the slicer entry primarly. If this is not available (in case the use did not put it the custom start g_code), then fall back to use the variable instead

Reply:
I can give it back to True if you prefer?

Reply:
No, just do it in two steps :

```
{ if param of slicer exist }
  USE PARAM OF SLICER
{ else }
  FALLBACK TO THE USE OF THE VARIABLE
```